src/multiboot-headers.md

```diff
@@ -282,7 +282,21 @@ Again, we’ll use math to let the computer calculate the sum for us. We add up
 the magic number, the mode code, and the header length, and then subtract it
 from a big number. `dd` then puts that value into this spot in our file.
 
-TODO: https://github.com/intermezzOS/book/issues/28
+> By the way...
```

Comment:
Hmmm... I'm still not seeing it checked in that way.
<img width="245" alt="screen shot 2016-01-12 at 14 54 53" src="https://cloud.githubusercontent.com/assets/1327285/12265279/825f166c-b93c-11e5-9663-5280c3b9ed9a.png">

Could it be you're looking at another aside?